code/mathematical_algorithms/src/primality_tests/fermat_primality_test/fermat_primality_test.py

```diff
@@ -0,0 +1,13 @@
+import random
+
+def isPrime(n, k=10):
+    if n <= 1:
+        return False
+    if n <= 3:
+        return True
+
+    for i in range(k):
+        a = random.randint(2, n-1)
```

Comment:
Since you're only using `randint`, you might as well just import that instead of the whole `random` module.